arangod/Pregel/Conductor/Conductor.cpp

```diff
@@ -307,35 +318,32 @@ bool Conductor::_startGlobalStep() {
 
 // The worker can (and should) periodically call back
 // to update its status
-void Conductor::workerStatusUpdate(VPackSlice const& data) {
+void Conductor::workerStatusUpdate(StatusUpdated&& update) {
```

Comment:
Is there a particular reason why you use rvalue-refs here (but as far as I saw) nowhere else?

Reply:
Yes, because `updateWorkerStatus` is called in this function, which requires an rvalue-ref for the status. I could of  course use `workerStatusUpdate(StatusUpdate const& update)`, and copy `update` but I thought this would be more convenient.
I guess nothing speaks against using rvalue-refs for all messages that are given to a function in conductor or worker.